src/components/Day/TimeDragSection/TimeBlocks.tsx

```diff
@@ -1,21 +1,49 @@
+import { useState } from 'react';
+import useDragBlock from 'src/hooks/useDragBlock';
 import { theme } from 'src/styles/theme';
 import { getTimeArray } from 'src/utils/timeArray';
 import styled from 'styled-components';
 
+import TimeBlock from './TimeBlock';
+
 interface timeType {
-  hour: number;
-  min: number;
+  hour: string;
+  min: string;
 }
 
 const LAST_MINIT_OF_HOUR = 45;
 
 function TimeBlocks() {
   const { timeArr } = getTimeArray();
+  const [isDragging, setIsDragging] = useState(false);
+  // const [isPlus, setIsPlus] = useState(true);
+  const [isExpected] = useState(false);
+  const [startBlock, setStartBlock] = useState('');
+  const [endBlock, setEndBlock] = useState('');
+
+  const handleDragState = (isDragging: boolean, startBlock: string, endBlock: string) => {
+    setIsDragging(isDragging);
+    if (startBlock !== '') {
+      setStartBlock(startBlock);
+    }
+    if (endBlock !== '') {
+      setEndBlock(endBlock);
+    }
+  };
```

Comment:
갠취..일수도 있지만 이 함수의 인자를 객체로 받으면 이 함수를 사용하는 곳에서 인자를 보낼 때 좀 더 명확할 것 같습미다 ! 
```suggestion
// dargStageParams 타입 지정 갈겨 
  const handleDragState = ({isDragging, startBlock, endBlock} : dargStageParams) => {
    setIsDragging(isDragging);
    if (startBlock !== '') {
      setStartBlock(startBlock);
    }
    if (endBlock !== '') {
      setEndBlock(endBlock);
    }
  };
```

Reply:
오오 좋네여!!!! 감사함다~